app/user_code_server/pyproject.toml

```diff
@@ -9,7 +9,7 @@ packages = [ { include = 'chainsail' } ]
 chainsail-user-code-server = 'chainsail.user_code_server:run'
 
 [tool.poetry.dependencies]
-python = "^3.8"
+python = "^3.9"
```

Comment:
Not a big deal, but just for documentation purposes was there a specific need for upgrading the Python version?

Reply:
See the commit message of https://github.com/tweag/chainsail/pull/480/commits/5f62efe1c3955f19bf693f03508686c57a2bd6dc: BridgeStan requires Python 3.9 and without that version bump, Poetry can't resolve the dependencies when installing BridgeStan. I'll add a comment in the `pyproject.toml` itself, too :slightly_smiling_face:

Reply:
Ah I missed that. Makes sense :+1: